src/includes/platforms/configuration/integrations/default.mdx

```diff
@@ -89,21 +89,23 @@ Available options:
 
 _Import name: `Sentry.Integrations.LinkedErrors`_
 
-This integration allows you to configure linked errors. They’ll be recursively read up to a specified limit, and lookup will be performed by a specific key. By default, the Sentry SDK sets the limit to five and the key used is `"cause"`.
+This integration allows you to configure linked errors. They'll be recursively read up to a specified limit, and lookup will be performed by a specific key on the captured Error object. By default, the integration records up to five errors and the key used for recurstion is `"cause"`.
 
 Available options:
 
 ```javascript
 {
-  key: string;
-  limit: number;
+  key: string; // default: "cause"
+  limit: number; // default: 5
 }
 ```
 
 Here is a code example of how this could be implemented:
 
 <PlatformContent includePath="configuration/linked-errors" />
 
+Additionally, the `LinkedErrors` integration enhances the capturing of [AggregateErrors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) by recursively iterating over the `errors` property on Error objects.
```

Comment:
> Does this reword make sense? This sentence didn't make sense to me as written. Does iterating over the errors property mean that the integration won't use the given key to recurse?

I originally had something similar but I was actually very specific with my wording because this integration doesn't capture any errors. It just enhances already captured error events with additional data.

> Could we move this up higher, above available options, so it's not missed?

For sure!

Reply:
Ahh, I see. 

Maybe we could phrase it more like how you explained it to me here then:

```suggestion
This integration also enhances captured error events with data from an [AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) where applicable, by recursively iterating over the `errors` property on Error objects.
```